src/pages/Home.jsx

```diff
@@ -1,42 +1,69 @@
 import { useDispatch, useSelector } from 'react-redux';
+import { CircularProgressbar } from 'react-circular-progressbar';
+import { useState } from 'react';
 import Form from '../components/Form';
+import 'react-circular-progressbar/dist/styles.css';
 import './home.css';
 import { deleteBook, deleteAPIBook } from '../components/redux/books/booksSlice';
 
 // Import the initialState and display
 export default function Home() {
   const dispatch = useDispatch();
   const books = useSelector((state) => state.book.books);
+  const [percentage, newPercentage] = useState(65);
 
+  const handlePercentage = (e) => {
+    e.preventDefault();
+    newPercentage(Math.floor(Math.random() * 100));
+  };
   return (
     <div className="bookList">
       { (books ?? []).filter((b) => b.item_id).map((book) => (
         <div className="eachBook" key={book.item_id} id={book.item_id}>
           <div className="left-pan">
             <h4>{book.category}</h4>
             {' '}
-            <h1>{book.author}</h1>
+            <h1>{book.title}</h1>
             {' '}
-            <p className="booktitle">{book.title}</p>
+            <p className="booktitle">{book.author}</p>
             {' '}
             <div className="buttonDiv">
               {' '}
               <button type="button">Comments</button>
-              <button className="remove" type="button" onClick={() => { dispatch(deleteBook(book.item_id)); dispatch(deleteAPIBook(book.item_id)); }}>REMOVE</button>
+              <button className="remove" type="button" onClick={() => { dispatch(deleteBook(book.item_id)); dispatch(deleteAPIBook(book.item_id)); }}>Remove</button>
               <button type="button">Edit</button>
             </div>
           </div>
```

Comment:
As it's a project requirement to use the same design and styles as used in the ZEPLIN template so please consider using the same set of styles. In design there is a vertical line between the items kindly consider adding it as used in Zeplin Design.

## YOUR DESIGN:-

![image](https://user-images.githubusercontent.com/94223367/230605557-3c200bba-48ac-49cb-9464-725c503174e1.png)



## ZEPLIN DESIGN:-

![image](https://user-images.githubusercontent.com/94223367/230605573-bfebcf61-810e-407c-80e8-9c89bf76ae63.png)